ports/mimxrt/Makefile

```diff
@@ -66,6 +66,7 @@ CFLAGS += -Wno-error=unused-parameter
 
 INC += -I.
 INC += -Ihal
+INC += -Iboards
```

Comment:
you won't need this line if you use `-DCLOCK_CONFIG_H='<boards/$(MCU_SERIES)_clock_config.h>'` below

Reply:
OK.